### ISSUE TITLE
os x install (brew postgresql 14): permission denied error

### ISSUE THREAD

**Issue by chrislungu** [pgaudit/pgaudit#234]
I'm trying to install pgaudit on my mac. I'm running pgsql 14 (installed using homebrew). I followed the instructions and used the REL_14_STABLE branch, as instructed, but I'm getting the following error:

    sudo make install USE_PGXS=1 PG_CONFIG=/user/local/var/postgresql@14/postgresql.cmd

    make: execvp: /usr/local/postgresql@14/postgresql.conf: Permission denied
    make: *** No rule to make target 'install'. Stop.

any thoughts? or does pgaudit only work with red hat linux? thanks!

**Comment by dwsteele** [comment]
If you figured it out it could you post the solution here? Might be helpful to somebody else in the future.

**Comment by chrislungu** [comment]
Sure thing.

What I did wrong was to try to run make using the postgresql config file instead of the pg_config binary. Once I changed that, it all worked like a charm. The following code works (tested) on os x (13.6) using a typical postgresql 14 installation using homebrew.

    git clone https://github.com/pgaudit/pgaudit.git
    cd pgaudit
    git checkout REL_14_STABLE

    make install USE_PGXS=1 PG_CONFIG=/usr/local/Cellar/postgresql@14/14.11_1/bin/pg_config

    create extension pgaudit;

to find out where your pg_config bin is, use "brew info postgresql" and suffix with /bin/pg_config. don't forget to add pgaudit to your "shared_preload_libraries" parameter in the conf file. to find out where your conf file is, run "show config_file;" in SQL (e.g. pgadmin).
